zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
+import requests
+import json
+import socket
+
+class JoinmeHandler(object):
+    def usage(self) -> str:
+        return '''
+         A bot that will post a link to joinme call. Mention the bot and it will give out instructions.
+        '''
+
+    def handle_message(self, message: Any, bot_handler: Any) -> None:
+        if 'ZISSHUDBUNIq' not in message['content']:

```

Comment:
Can you add  a comment what this check accomplishes (in particular, what does "ZISSHUDBUNiq" mean?